Source/WebCore/rendering/style/StyleGradientImage.h

```diff
@@ -34,9 +34,8 @@
 namespace WebCore {
 
 struct StyleGradientImageStop {
-    RefPtr<CSSPrimitiveValue> color;
+    std::optional<StyleColor> color;
```

Comment:
Both `Color` and `StyleColor` have "invalid" values, so I was hoping we’d almost never use `std::optional<StyleColor>` just as we wouldn’t use `std::optional<RefPtr<Node>>` since `nullptr` is there and handy. On the other hand, doing it explicitly like this is so nice. Maybe some day we could change `StyleColor` and `Color` to not have invalid states when used directly, and only have them when combined with a template analogous to `std::optional`; lets us be explicit about whether there is a null, but without extra storage overhead.